pkg/operator/ceph/cluster/controller.go

```diff
@@ -529,6 +564,27 @@ func (c *ClusterController) onUpdate(oldObj, newObj interface{}) {
 		}
 		return
 	}
+
+	// Display success after upgrade
+	if versionChanged {
+		versions, err := client.GetCephVersions(c.context, cluster.Namespace)
+		if err != nil {
+			logger.Errorf("failed to get ceph daemons versions. %+v", err)
+			return
+		}
+
+		if len(versions.Overall) == 1 {
```

Comment:
Since we are already doing this check, if this is not equal to 1, we should probably log an error/warning message saying that the upgrade completed but that we somehow still have more than one version running.